core/src/main/java/io/questdb/log/LogRollingFileWriter.java

```diff
@@ -108,6 +118,23 @@ public void bindProperties(LogFactory factory) {
             nRollSize = Long.MAX_VALUE;
         }
 
+        if(this.sizeLimit != null) {
```

Comment:
Please reformat the code properly. The simplest way to do that is to use IDEA's autoformatter: https://github.com/questdb/questdb/blob/master/CONTRIBUTING.md#code-formatting
```suggestion
        if (this.sizeLimit != null) {
```